spec/feature/add_a_space_spec.rb

```diff
@@ -0,0 +1,6 @@
+feature 'add a space: ' do
```

Comment:
Maybe add another test for actually filling out the form, submitting and testing that it takes you to a page with the correct path? (`expect(page).to have_current_path('/add-space')`) or something?